tfjs-core/src/ops/fused_test.ts

```diff
@@ -640,6 +655,568 @@ describeWithFlags('fused conv2d', ALL_ENVS, () => {
                          ]));
      });
 
+  it('relu stride 2 x=[1,8,8,16] f=[3,3,16,1] s=[2,2] d=4 p=same', async () => {
+    const inputDepth = 16;
+    const xSize = 8;
+    const inputShape: [number, number, number, number] =
+        [1, xSize, xSize, inputDepth];
+    const outputDepth = 4;
+    const fSize = 3;
+    const pad = 'same';
+    const stride: [number, number] = [2, 2];
+
+    const inputs = generateCaseInputs(
+        1 * xSize * xSize * inputDepth,
+        fSize * fSize * inputDepth * outputDepth);
+    const x = tf.tensor4d(inputs.input, inputShape);
+    const w =
+        tf.tensor4d(inputs.filter, [fSize, fSize, inputDepth, outputDepth]);
+
+    const result = tf.fused.conv2d({
+      x,
+      filter: w,
+      strides: stride,
+      pad,
+      dataFormat: 'NHWC',
+      dilations: [1, 1],
+      activation: 'relu'
+    });
+    expect(result.shape).toEqual([1, 4, 4, 4]);
+    expectArraysClose(
+        await result.data(), new Float32Array([
```

Comment:
Would it be possible to reduce the number of test cases to 2? 

Also, these tests will probably fail in Safari due to overflow issues in WebGL (see: https://github.com/tensorflow/tfjs/issues/3143). While we work on fixing the underlying issue, would you mind changing your input generator so that the outputs stay under ~1000?